backend/clap/src/main/resources/templates/reminder_template.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Przypomnienie: Wystawienie oceny</title>
+</head>
+<body style="font-family: Arial, sans-serif;">
+
+<div style="max-width: 600px; margin: 0 auto; padding: 20px; background-color: #f8f8f8; border: 1px solid #ddd; border-radius: 5px;">
+
+    <h5 style="color: #333"> For English, scroll down. </h5>
+
+    <!-- Polish -->
+    <h2 style="color: #333;">Przypomnienie: Wystawienie oceny</h2>
+    <p>
+        Szanowny asesorze [AssessorName] [AssessorLastname],
+    </p>
+    <p>
+        Termin wystawienia oceny [RatingType] do projektu [SubmissionName] mija [FinishDate]. [IsCreatedPl]
+    </p>
+    <p>
+        Pozdrawiamy,<br>
```

Comment:
It's missing information that `this email was generated automatically` right? Or does spring add it by itself